.github/pull_request_template.md

```diff
@@ -0,0 +1,43 @@
+## Summary
+
+<!-- 
+    Provide an overview of what this pull request aims to address or achieve.
+-->
+
+**Related issue
+**: <!-- Add the issue number in format [#<number>](link) or set to "None" if this is not related to a reported issue. -->
```

Comment:
Is this formated correctly?
** would bold text but there is new line before closing it

Reply:
Hmmm don't know how this got reformatted. Thanks, fixed now!